src/main/java/com/puppycrawl/tools/checkstyle/JavadocDetailNodeParser.java

```diff
@@ -443,6 +465,57 @@ private static String getNodeClassNameWithoutContext(ParseTree node) {
     }
 
     /**
+     * Method to search for a missed HTML tag which could possibly be responsibe for ANTLR error.
+     * @param exception {@code NoViableAltException} object catched while parsing javadoc
+     * @return returns appropriate {@code ParseErrorMessage} if a HTML close tag is missed;
+     *     null otherwise
+     */
+    private ParseErrorMessage generateMissedHtmlTagErrorMessage(RecognitionException exception) {
```

Comment:
javadoc says that method searches for a tag, but method name says that it generates a message.